css/order.css

```diff
@@ -0,0 +1,227 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* font-family: 'Montserrat', sans-serif;
+font-family: 'M PLUS 1p', sans-serif;
+font-family: 'Work Sans', sans-serif; */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: 0;
+  border: none;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  text-align: center;
+  color: #666666;
+}
+
+.form-section {
+  width: 100%;
+  display: flex;
+}
+
+.image-container {
+  width: 50%;
+}
+
+.form-image {
+  height: auto;
+  width: 100%;
+}
+
+.form-container {
+  width: 50%;
+  font-family: "M PLUS 1p", sans-serif;
+  align-items: flex-start;
+  text-align: start;
+  padding: 4rem 6rem;
+}
+
+.form-container h1 {
+  color: #de6e49;
+  font-size: 2.1rem;
+  font-weight: 600;
+}
+
+.form-container input {
+  border: 1px solid grey;
+  border-radius: 4px;
+}
+
+.borders label {
+  display: flex;
+  margin-top: 18px;
+}
```

Comment:
What does the `.borders` class do? It doesn't seem to be declared anywhere. Perhaps you could rename it and apply it to the `label` elements directly.